src/frontend/src/app/scheduler/list-tasks/list-tasks.component.html

```diff
@@ -25,8 +25,8 @@
       <tr *ngFor="let task of tasksData">
         <td><small>{{task.process_name}}</small></td>
         <td><small>{{task.state}}</small></td>
-        <td><small>{{task.start_time != 'None' ?(task.start_time | date: 'dd-MM-yyyy HH:mm:ss') : task.start_time}}</small></td>
-        <td><small>{{task.end_time != 'None'?(task.end_time | date: 'dd-MM-yyyy HH:mm:ss') : task.end_time}}</small></td>
+        <td><small>{{task.start_time != 'None' ?(task.start_time | moment: 'DD/MM/YYYY HH:mm:ss') : task.start_time}}</small></td>
+        <td><small>{{task.end_time != 'None'?(task.end_time | moment: 'DD/MM/YYYY HH:mm:ss') : task.end_time}}</small></td>
```

Comment:
in case of end_time="" it gives Invalid date;  (running tasks)

```
{"tasks": [{"state": "COMPLETE", "process_name": "purge", "exit_code": "0", "pid": "9557", "end_time": "2017-11-29 15:48:24.904505+05:30", "id": "af4faefd-c4ff-481d-ac5c-edb913aad569", "reason": "", "start_time": "2017-11-29 15:48:24.222508+05:30"}, {"state": "RUNNING", "process_name": "sending process", "exit_code": "", "pid": "9652", "end_time": "", "id": "f77253e6-da8e-41a1-a1ec-39d1e0046595", "reason": "", "start_time": "2017-11-29 15:51:09.234475+05:30"}, {"state": "RUNNING", "process_name": "statistics to pi", "exit_code": "", "pid": "9660", "end_time": "", "id": "3f8f014c-1f49-4018-a8eb-61f2a9f34fa4", "reason": "", "start_time": "2017-11-29 15:51:19.220856+05:30"}, {"state": "COMPLETE", "process_name": "stats collector", "exit_code": "0", "pid": "9665", "end_time": "2017-11-29 15:51:24.809084+05:30", "id": "88f54970-057d-477e-ba31-952f634c60ca", "reason": "", "start_time": "2017-11-29 15:51:24.221095+05:30"}]} 
```